### DISCUSSION TITLE
蓝灯的安卓版无法使用，但pc版可以用

### DISCUSSION THREAD

**Discussion by ghost6614** [getlantern/lantern-forum-cn#79] [Q&A]
这是什么情况，而且安卓版无法升级成专业版

**Comment by 13600100046** [comment]
比我的好一点，或者你可以试试退出登录，看看免费版的行不行，我是专业版就用不了，免费版还能用

**Reply by ghost6614** [reply]
都不行

**Comment by MaxMKMax** [comment]
@ghost6614 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

在这之前，您方便让我知道您现在用的是哪个版本吗？打个比方version xxx.xxx . 然后用安卓手机还是萍果手机? 或者哪款的电脑呢？

然后, 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

我们将非常感谢您。

**Reply by ghost6614** [reply]
手机是安卓的huawei mate60，电脑是华硕的天选

**Reply by ghost6614** [reply]
版本是7.6.15（20231019.000631）

**Reply by panrupeng** [reply]
我也是同样的问题，电脑上可以，安卓手机上不行，版本同样时7.6.15，手机是小米14pro

**Reply by f782175097** [reply]
换了新手机后无法打开lantern软件，提示无法兼容32位,经查是由于今年新一代的cpu例如骁龙8-3,天矶9300已不支持32位的应用了(震惊，lantern安卓端居然是32位应用？)

**Comment by MaxMKMax** [comment]
@ghost6614 如果您已付款。可以将您的问题发送至 support@getlantern.org
需要包含购买证明/ 收据和8位数单号和您购买 Lantern 时用的电子邮件。然后描述您遇到的具体问题。我们会进行处理。谢谢!

**Reply by ghost6614** [reply]
我用了快一年了，最近安卓版上不去了，我就卸载重装一下，然后就想通过PC端授权手机端，PC端显示授权成功，但手机端没有反应。

**Reply by MaxMKMax** [reply]
@ghost6614 非常抱歉影响您的体验感。我们正在努力优化了，很快就会修复好。请再给我们一点点时间。真的非常抱歉。

**Comment by keene06** [comment]
一样的问题，专业版手机端不能使用，PC正常

**Reply by MaxMKMax** [reply]
@keene06 非常抱歉影响您的体验感。我们正在努力优化了，很快就会修复好。请再给我们一点点时间。真的非常抱歉。

**Comment by Ivanzhanglm** [comment]
我还以为只有我一个人的不好使，原来都这样了

**Comment by MaxMKMax** [comment]
@Ivanzhanglm 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

在这之前，您方便让我知道您现在用的是哪个版本吗？打个比方version xxx.xxx . 然后用安卓手机还是萍果手机? 或者哪款的电脑呢？

然后, 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
我们将非常感谢您。

**Reply by Ivanzhanglm** [reply]
magic os 8 ,彻底无法使用了，直接提示不兼容了。

**Reply by MaxMKMax** [reply]
@Ivanzhanglm 您好，谢谢您的反馈。您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。之后返回给我 8 位数report/log 编号。提交问题报告后，可以在您的电子邮件中找到report/log编号。谢谢。

**Comment by Ivanzhanglm** [comment]
软件版本7.7.0（ 20231206.064213  ），手机荣耀magicos7.2.0.136

Mike K ***@***.***> 于2024年1月26日周五 17:47写道：

**Comment by zwdwxc** [comment]
我的也是这样，今天重新安装了手机，结果手机就用不了蓝灯，不管是免费版，还是专业版。电脑正常。
手机vivo，安装的蓝灯是7.7.1（20240107）。

**Reply by MaxMKMax** [reply]
@zwdwxc 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
我们将非常感谢您。
